### ISSUE TITLE
Artists name potentially causing issue with get_artist_audio_features()

### ISSUE THREAD

**Issue by delabj** [charlie86/spotifyr#99]
This is likely a corner case issue, but when using get_artist_audio_features() For the artist Nas,  by

```R
get_artist_audio_features(artist="nas")
```
I get the following error. 

```R
Error in UseMethod("rename_") : 
  no applicable method for 'rename_' applied to an object of class "list"
```

I'm not sure why this causes this error as the artist does have albums. But it does work when I use the featured_on with include_groups.

**Comment by neilcollins2018** [comment]
I'm not sure about the above case but this error occurs for me when the artist searched for does not have any existing albums. For me it comes up in this section of code from the function:

```

   if (num_loops_artist_albums > 1) {

      artist_albums <- map_df(1:num_loops_artist_albums, function(this_loop) {
        get_artist_albums(artist_id, include_groups = include_groups, 
                        offset = (this_loop - 1) * 20, authorization = authorization)
      })  } else {
      artist_albums <- artist_albums$items  
}
  artist_albums <- artist_albums %>% 
                              rename(album_id = id,  album_name = name) %>% 
      mutate(album_release_year = case_when(
        release_date_precision == "year" ~ suppressWarnings(as.numeric(release_date)), 
        release_date_precision == "day" ~ year(as.Date(release_date, "%Y-%m-%d", origin = "1970-01-01")), 
        TRUE ~ as.numeric(NA)))

```

**Comment by mswatek** [comment]
I'm having the same issue when trying to pull a large list of artists. Anyone figure out why this is happening?

**Comment by delabj** [comment]
@mswatek I think the issue might be what @neilcollins2018 mentioned. I've put the side project I was using this for on hold, but I did find that my specific example, didn't have albums per se, but singles and other non album versions. 

At the time I managed to get this to work with just this
`get_artist_audio_features(artist="nas", include_groups	=c("album" ,"single" ,"appears_on" ,"compilation" )`

One thing I ended up doing ( I was analyzing my music library) was to put the tracks of interest in a playlist. This worked really well, and I was able to get the artist ID

**Comment by antaldaniel** [comment]
I think that this is the same issue as [109](https://github.com/charlie86/spotifyr/issues/109). Will look into making a warning here.  PR's are apprectiated.

**Comment by antaldaniel** [comment]
This is also the same as [159](https://github.com/charlie86/spotifyr/issues/159)

**Comment by delabj** [comment]
Yep, I probably should have come back once I figured this out. This might be more of a documentation type of thing? I was thinking of albums as a broader term but spotify uses this a bit more specifically. Do you want me close this issue since it's also covered in #159 ?
